src/main/java/com/bravo/user/service/PaymentService.java

```diff
@@ -0,0 +1,46 @@
+package com.bravo.user.service;
+
+import com.bravo.user.dao.model.Payment;
+import com.bravo.user.dao.model.mapper.ResourceMapper;
+import com.bravo.user.dao.repository.PaymentRepository;
+import com.bravo.user.dao.specification.PaymentSpecification;
+import com.bravo.user.model.dto.PaymentDto;
+import com.bravo.user.model.filter.PaymentFilter;
+import com.bravo.user.utility.PageUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.stereotype.Service;
+import javax.servlet.http.HttpServletResponse;
+import java.util.List;
+
+@Service
+public class PaymentService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(PaymentService.class);
```

Comment:
I've seen this in many different ways so would like your thoughts:  here, LOGGER is all caps. Is that in line with established naming conventions?

Reply:
Existing UserService.class used this format, so I kept it consistent for the PaymentService.    I've used the SLF4J annotation previously (so the reference is log.info, etc.).    If no prior precedent in existing code, I would have likely named it log.